src/Visualization/Python/Render1D.py

```diff
@@ -0,0 +1,277 @@
+
+
+def get_data(args):
+    '''
+    Get the data to be plotted
+    :return: the array of data, coords, and time
+    '''
+    time = []
+    coords = []
+    data = []
+    file_prefix = args["file_prefix"]
+    filename_list = args["filename_list"]
+    var_name = args["var"]
+    h5files = get_h5_files(file_prefix, filename_list)
+    volfiles = [h5file['element_data.vol'] for h5file in h5files]
+    # Get a list of times from the first vol file
+    ids_times = [(obs_id, volfiles[0][obs_id].attrs['observation_value'])
+                 for obs_id in volfiles[0].keys()]
+    ids_times.sort(key=lambda pair: pair[1])
+    time_data = {}
+    for id_and_time in ids_times:
+        time_data[id_and_time] = []
+        local_data = []
+        local_coords = []
+        for volfile in volfiles:
+            obs_id_file = volfile[id_and_time[0]]
+            local_data += list(obs_id_file[var_name])
+            local_coords += list(obs_id_file['InertialCoordinates_x'])
+        local_coords_and_data = zip(local_coords, local_data)
+        time_data[id_and_time] += local_coords_and_data
+
+        time_data[id_and_time].sort(key=lambda pair: pair[0])
+        time.append(id_and_time[1])
+        coords.append([pair[0] for pair in time_data[id_and_time]])
+        data.append([pair[1] for pair in time_data[id_and_time]])
+
+    for h5file in h5files:
+        h5file.close()
+    return time, coords, data
+
+
+def render_single_time(args, coords, data):
+    '''
+    Renders a single time snap shot
+    :param args: the command line arguments
+    :return: None
+    '''
+    plt.xlabel("x")
+    plt.ylabel(args['var'])
+    plt.title("t = %.5f" % int(args['time']))
+    plt.plot(coords[int(args['time'])], data[int(args['time'])], 'o')
+    if args['save']:
+        print("Writing still to file %s.pdf" % args['save'])
+        plt.savefig("%s.pdf" % args['save'],
+                    transparent=True,
+                    format='pdf',
+                    bbox_inches='tight')
+    else:
+        plt.show()
+    return None
+
+
+def render_animation(args, time, coords, data):
+    '''
+    Render an animation of the data
+    :param args: the command line arguments
+    :return: None
+    '''
+    def init():
+        '''
+        Initialize the animation canvas
+        :return: empty line and title info
+        '''
+        line.set_data([], [])
+        title.set_text("")
+        return line, title
+
+    def animate(iteration):
+        '''
+        Update the animation canvas
+        '''
+        title.set_text("t = %.5f" % time[iteration])
+        line.set_data(coords[iteration], data[iteration])
+        return line, title
+
+    fig = plt.figure()
+    ax = plt.axes(xlim=(find_extrema_over_data_set(coords)),
+                  ylim=(find_extrema_over_data_set(data)))
+
+    line, = ax.plot([], [], 'o', lw=2)
+    ax.set_xlabel('x')
+    ax.set_ylabel(args['var'])
+    title = ax.set_title("")
+    my_interval = float(args['interval'])
```

Comment:
You're not using the `fps` from the command line args. Either get rid of the `interval` argument and use `fps` here
```suggestion
    interval_in_ms = 1000 / fps
```
or the other way round, i.e. get rid of the `fps` argument.